modules/ROOT/pages/java-support.adoc

```diff
@@ -0,0 +1,134 @@
+= Java Support
+
+MuleSoft is adopting Java’s long-term-support (LTS) release cadence, beginning with Mule Runtime 4.6, which adds support for Java 17 LTS. 
+
+Many third-party vendors are discontinuing support for Java 8, which impacts the MuleSoft product stack. Adopting this new release cadence provides greater security, compliance, and performance.
+
+This change has no immediate impact to Mule applications and Mule Gateway policies and proxies that are running on currently supported Mule versions, or on applications that are upgraded to Mule Runtime 4.6, as existing Java 8 apps will continue to run on Java 8 until February 2026, giving you time to plan for your upgrade to Java 17.
+
+Java 17 upgrade planning includes evaluating connectors, integration apps, Mule Gateway policies, and proxies, as all of these components rely on Java. 
+
+== Mule Runtime Java Support
+
+The following table shows the timeline for Java 8 and Java 17 support:
+
+[%header%autowidth.spread]
+|===
+|Mule Version| Release Date |Java Version | End of Standard Support | End of Extended Support
+|4.5 Edge | October 2023 |8 | February 2024 | June 2024
+|4.6 LTS | February 2024 | 8 and 17 | February 2025 | February 2026
+|4.6 Edge | February 2024 |8 and 17| June 2024 | October 2024
+|4.7 Edge | June 2024 | 8 and 17 | October 2024 | February 2026 ?
+| 4.8 Edge | October 2024 | 8 and 17 | February 2025 | June 2025
+| 4.9 LTS | February 2025 | 17 | February 2026 | February 2027
+| 4.9 Edge | February 2025 | 17 | June 2025 | October 2025
+|===
+
+[NOTE]
+Along with the end of support for Java 8 with the Mule Runtime 4.9 release in February 2025, Java 11 will also no longer be available. Plan your upgrade path for apps that are running on both Java 8 and Java 11.
+
+
+== Upgrading Mule Apps
```

Comment:
I wonder if we need to have at least a small callout for RTM, CloudHub, CloudHub 2.0, RTF. Like even just linking to the RNs because these products have some J17 changes right?